apps/wallet-dashboard/components/Dialogs/SendToken/views/SentSuccessView.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2024 IOTA Stiftung
```

Comment:
I'm thinking if this view is necessary. Why not use `TransactionDialogDetails` in `SendTokenDialog`?